core/src/test/scala/integration/kafka/api/SaslPlainPlaintextConsumerTest.scala

```diff
@@ -29,14 +28,16 @@ class SaslPlainPlaintextConsumerTest extends BaseConsumerTest with SaslSetup {
   private val kafkaServerJaasEntryName =
     s"${listenerName.value.toLowerCase(Locale.ROOT)}.${JaasTestUtils.KafkaServerContextName}"
   this.serverConfig.setProperty(KafkaConfig.ZkEnableSecureAclsProp, "false")
+  // disable secure acls of zkClient in ZooKeeperTestHarness
+  override protected def zkAclsEnabled = Some(false)
```

Comment:
why def?

Reply:
This is related above comment. This is where I need to override the zkAclsEnabled.